src/evm/engine_db/tycho_db.rs

```diff
@@ -230,6 +230,15 @@ impl PreCachedDB {
             .as_ref()
             .map(|header| header.number)
     }
+
+    /// Clear all state from the database.
+    ///
+    /// This is useful for test isolation to ensure state doesn't leak between tests.
```

Comment:
I think it's irrelevant to comment here that this is done to avoid leaking between tests - it's not the function's purpose and it can be used in other cases.